schema/definitions/0.8.0/schema/fmu_results.json

```diff
@@ -1370,7 +1369,31 @@
                         "ssdl"
                     ]
                 }
-            }
+            },
+            "allOf": [
```

Comment:
Is `allOf` needed when only one element in the list? Possibly relevant: https://json-schema.org/understanding-json-schema/reference/conditionals.html#id6

Reply:
You're right, `allOf` is not needed 👍